react/week4/githubsearcher/src/components/Search.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import { Context } from '../context';
+import Users from './User';
+import Loading from './Loading';
+
+class Search extends Component {
+   state = {
+      usersArray: [],
+      userName: '',
+   };
+
+   findUser = (e) => {
+      e.preventDefault();
+      fetch(`https://api.github.com/search/users?q=${this.state.userName}`)
+         .then((res) => res.json())
+         .then((data) => {
+            console.log(data.items);
+            this.setState({
+               usersArray: data.items,
+               userName: '',
+            });
+         })
+         .catch((error) => console.log(error));
+   };
+
+   onChange = (e) => {
+      this.setState({ userName: e.target.value });
+   };
+   render() {
+      return (
+         <Context.Consumer>
+            {(value, index) => {
```

Comment:
You declare `value` here but you never use it. The user list should be coming from it, not from the internal state. See comment above.